### PR TITLE
fix: add milkshakegum

### LINKED ISSUE #1381: Milkshakegum
#### Profile Screenshot

github page

### PR DESCRIPTION
**Pull request by milkshakegum**
resolves #1381

## Checklist for profile addition

- [X] Name the PR `fix: add milkshakegum`.
- [X] Added file named `milkshakegum.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [X] You've linked the PR to the correct issue.

### CODE REVIEW

File: profiles/milkshakegum.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/29112173/190592868-c5f1d4a3-5d60-48d3-8e60-f28dce110ba4.png",
+    "issueId": 66,
```
Comment (Panquesito7) [inline profiles/milkshakegum.json:3]:
```suggestion
    "issueId": 1381,
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

File: profiles/milkshakegum.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "name": "Milk",
+    "username": "milkshakegum"
+  }
```
Comment (Panquesito7) [inline profiles/milkshakegum.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**